README.md

```diff
@@ -1,70 +1,137 @@
+the `spec` sub-resource describes the desired state that CVO should achieve and
+maintain, and the `status` sub-resource that describes the state observed by CVO
+and its current status.
 
-1. Make sure you have `oc` binary from https://github.com/openshift/origin master as it requires `adm release` subcommand.
+In a typical OpenShift cluster, there will be a cluster-scoped `ClusterVersion`
+resource called `version`:
 
-2. Run the following command to create release-image at `docker.io/abhinavdahiya/origin-release:latest`:
-
-```sh
-oc adm release new -n origin \
-    --from-image-stream=origin-v4.0 \
-    --to-image-base=docker.io/abhinavdahiya/origin-cluster-version-operator:latest \
-    --to-image docker.io/abhinavdahiya/origin-release:latest
+```console
+$ oc get clusterversion version
+NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+version   4.11.3    True        False         85d     Cluster version is 4.11.3
 ```
 
-## Installing CVO and operators in cluster.
+Note that as a user or a cluster administrator, you usually do not interact with
+the `ClusterVersion` resource directly but via either the [`oc adm upgrade`](https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/administrator-cli-commands.html#oc-adm-upgrade) CLI
+or the [web console](https://docs.openshift.com/container-platform/latest/updating/updating-cluster-within-minor.html#update-upgrading-web_updating-cluster-within-minor).
 
-1. Use CVO `render` to render all the manifests from release-payload to a directory. Refer the [Using CVO to render the release-image locally](#using-cvo-to-render-the-release-image-locally) section for more information.
+## Understanding Upgrades
 
-2. Create the operators from the manifests by using `oc create -f <directory when CVO rendered manifests>`.
+> :bulb: This section is only a high-level overview. See the [Update Process](https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/user/update-workflow.md) and [Reconciliation](https://github.com/openshift/enhancements/blob/master/dev-guide/cluster-version-operator/user/reconciliation.md) documents in the [dev-guide](https://github.com/openshift/enhancements/blob/master/dev-guide) for more details.
 
-## Using CVO to render the release-image locally
+The Cluster Version Operator continuously fetches information about upgrade
+paths for the configured channel from the OpenShift Update Service (OSUS). It
+stores the recommended update options in the `status.availableUpdates` field of
+its `ClusterVersion` resource.
 
-1. Run the following command to get render the release-payload contents to `/tmp/cvo`
+The intent to upgrade the cluster to another version is expressed by storing the
+desired version in the `spec.desiredUpdate` field, either manually by the
+administrator through the CLI or web console, or CVO does it automatically when
+automated upgrades are enabled.
```

Comment:
Probably drop the "automated upgrades" mention for now, because we don't expose any user-facing knobs for that (openshift/enhancements#124 floated a knob, but was deferred).  And when automatic updates are enabled, there will be a knob to ask the CVO to perform them, but the CVO won't implement it by touching `spec.desiredUpdate` (because the CVO and clusters admins competing for the same `spec` properties would be racy, so the CVO's ClusterVersion `spec` access is read-only).